ocrd/ocrd/workspace.py

```diff
@@ -6,12 +7,22 @@
 import numpy as np
 
 from ocrd_models import OcrdMets, OcrdExif
-from ocrd_utils import getLogger, is_local_filename, abspath
+from ocrd_utils import (
+    abspath,
+    coordinates_of_segment,
+    crop_image,
+    getLogger,
+    image_from_polygon,
+    is_local_filename,
+    polygon_from_points,
+    xywh_from_points,
+)
 
 from .workspace_backup import WorkspaceBackupManager
 
 log = getLogger('ocrd.workspace')
 
+
```

Comment:
Following up on the above comment on advertising the new functions, could we change the docstring of `resolve_image_as_pil` to point to them (and perhaps even warning of deprecation)?

Reply:
See 50f7621